examples/fabric/nested_ingestion_pipeline/README.md

```diff
@@ -0,0 +1,96 @@
+# Nested pipelines
+
+The following example, exemplifies the utilization of nested pipelines including control activities like ForEach and If.
+
+![Nested Pipelines](pl_main_pipeline.png)
+
+The main pipeline is composed of 2 top activities:
+
+- a lookup activity named "Read Configuration File
+- a ForEach activity names "ForEach"
+
+Within the ForEach activity, there is a nested If activity called "If New Or Updated" and if this activity evaluates to True, a Invoke Pipeline activity named "Invoke Ingestion Pipeline" is triggered.
+
+The Ingestion pipeline is composed also by two activities:
+
+- a Copy activity named "Copy NYCData to ADLS" that copies from the Web to an ADLS Gen2 storage,
+- a Copy activity named "CopyNYCData to Lakehouse" that copies from the ADLS Gen2 storage to the Lakehouse.
+
+![Ingestion Pipeline](pl_ingestion_pipeline.png)
+
+## Testing the pl_main activities
+
+The file that contains the individual activity tests for the pl_main_pipeline is located [here](./fabric/tests/test_pl_main_activity.py) and it contains three individual activity tests:
+
+- test_activity_read_configuration_file, which is responsible to test if the json configuration file is correctly located and defined.
+- test_activity_if, responsible to check id the if condition expression is well formed in order to decide wheather or not new or updated data should be ingested.
```

Comment:
```suggestion
- test_activity_if, responsible to check if the if condition expression is well formed in order to decide whether new or updated data should be ingested.
```

Reply:
Rewrote it to add more clarity. Also added the new case for the if that evaluates to True and for the if that evaluates to False